packages/cache/src/cache.ts

```diff
@@ -0,0 +1,23 @@
+import { getCachedAdapter, setCachedAdapter } from "./config";
+
+class Cache {
+  async get(key: string) {
```

Comment:
I think you need to implement the `set` and `remove` methods as well. I think you are already aware of that but just commenting so you do not forget

Reply:
> I think you need to implement the `set` and `remove` methods as well. I think you are already aware of that but just commenting so you do not forget

Yeap, as soon as I start the example for real I'm gonna implement the others methods, don't worry about it